scripts/drone/steps/lib.star

```diff
@@ -760,6 +760,21 @@ def build_docs_website_step():
         ],
     }
 
+def build_swagger_api_step():
```

Comment:
since we need to add a commit into grafana repostiory, maybe instead of drone, we need to use github actions like we did for backport, it would be more convinient to open a new PR there